src/components/LogUpdate/CriticalCarePreview.tsx

```diff
@@ -49,14 +49,26 @@ export default function CriticalCarePreview(props: Props) {
 
   return (
     <div className="w-full transition-all duration-200 ease-in-out md:mx-auto md:max-w-5xl md:pt-8">
-      <div className="py-4">
+      <div className="py-4 flex items-center justify-between max-[368px]:flex-col max-[368px]:items-start">
```

Comment:
if it's for mobile view use `md:` modifier instead

```suggestion
      <div className="py-4 flex items-center justify-between md:flex-col md:items-start">
```

Reply:
@kihan2518B This is still unresolved btw

Reply:
Got this in last commit.